### DISCUSSION TITLE
browser.newtab.url script: How do I autofocus a new tab element instead of the firefox address bar?

### DISCUSSION THREAD

**Discussion by bash-ful** [xiaoxiaoflood/firefox-scripts#266] [General Support]
I want an input field be focused upon opening a new tab. It already has the `autofocus` attribute, but the cursor is still on the address bar by default. Is there a way to change this behavior?

I am using firefox version `115.0.2` on Linux Mint 21.1.

**Comment by marty60** [comment]
That's a good question, I wish someone had an easy answer.

What I did was create a script button to open a new tab without focus, just place it in the chrome folder with your other scripts. It's set to Startpage but If you open the script with notepad you can easily change it to whatever url you want. You can also change the base64 icon. 

  
[new tab.uc.js.zip](https://github.com/xiaoxiaoflood/firefox-scripts/files/12268354/new.tab.uc.js.zip)
